frontend/src/features/permits/hooks/hooks.ts

```diff
@@ -347,20 +344,3 @@ export const useAmendmentApplicationQuery = (originalPermitId?: string) => {
     refetchOnWindowFocus: false, // prevent unnecessary multiple queries on page showing up in foreground
   });
 };
-
-/**
- * A custom react query hook that fetches applications in progress.
- * @returns List of applications in progress
- */
-export const useApplicationsInProgressQuery = () => {
-  const applicationsInProgressQuery = useQuery({
```

Comment:
Why did you remove this hook?

Reply:
because we're not using a hook in the other paginated permit queries. I was trying to follow a similar pattern as the `Active Permits` and the `Expired Permits` tabs

Reply:
does using a hook impact the implementation/logic of the pagination vs not using a hook?

Reply:
because if it doesn't make a difference, then I feel like we should use the hook(s) for all the various tabs

Reply:
ok I guess we can leave it like this for now, but in the future if there's additional logic associated with querying we should probably move it into a hook instead.